frame/balances/src/tests_local.rs

```diff
@@ -50,11 +54,23 @@ parameter_types! {
 	pub const MaximumBlockLength: u32 = 2 * 1024;
 	pub const AvailableBlockRatio: Perbill = Perbill::one();
 }
+#[derive(Debug, Default)]
+pub struct Call {}
```

Comment:
Is this something that's going to be needed in almost every test file? This is a no go. We should either:
1. Implement `Dispatchable for ()` and make it a no-op.
2. Create a `testing::Call` type (enum with variants like: `NoOp, Panic`) that can be used to avoid duplication.

Reply:
> Implement Dispatchable for () and make it a no-op.

👍

Reply:
Regarding 1:
I added a `Dispatchable for ()` in this PR and it is used for almost all tests. Only the two instances in this test need a custom implementation because they use a `SignedExtension` with their dummy `Test` trait. Therefore the supplied call needs a `Info=PostDispatchInfo` bound.

Regarding 2:
Yes I can de-duplicate those two instances by have a  `trait TestCall<T>` where `T` is used as the `Info`.

Reply:
I de-duplicated the two occurrences.